CommandHelper/LocalPackages/commands/chars/deadmanbostonbrand.yml

```diff
@@ -0,0 +1,24 @@
+chatcolor: '&8'
+class: deadmanbostonbrand
+name: §4D§8e§fad§4m§fa§4n§8(§fBB§8)
```

Comment:
I just realized there is only one character named Deadman, so there is no reason to have his full name and initials